iterative/utils/logger.go

```diff
@@ -66,46 +69,51 @@ func (f *tpiFormatter) Format(entry *logrus.Entry) ([]byte, error) {
 
 	if message == "status" {
 		status := d.Get("status").(map[string]interface{})
-		terminatedStr := "terminated 🔵"
 
-		running := "queued 🟣"
-		if status["running"] != nil {
-			running = terminatedStr
-			if status["running"].(int) == 1 {
-				running = "running 🟡"
-			}
-		}
+		message = fmt.Sprintf("\x1b[%dmStatus: queued 🔵\x1b[0m", colors["DEBUG"])
 
-		success := ""
-		if running == terminatedStr {
-			success = "without any output"
-			if status["succeeded"] != nil && status["succeeded"].(int) == 1 {
-				success = "succesfully 🟢"
-			}
-			if status["failed"] != nil && status["failed"].(int) == 1 {
-				success = "with errors 🔴"
-			}
+		if status["succeeded"] != nil && status["succeeded"].(int) > 0 {
+			message = fmt.Sprintf("\x1b[%dmStatus: completed succesfully 🟢\x1b[0m", colors["SUCCESS"])
+		}
+		if status["failed"] != nil && status["failed"].(int) > 0 {
+			message = fmt.Sprintf("\x1b[%dmStatus: completed with errors 🔴\x1b[0m", colors["ERROR"])
+		}
+		if status["running"] != nil && status["running"].(int) > 0 {
+			message = fmt.Sprintf("\x1b[%dmStatus: running 🟡\x1b[0m", colors["WARNING"])
 		}
```

Comment:
This should be changed if we can trust the status. At the time of doing the logs that was not totally the case. Sometimes we had undesired effects and thats why the workarounds

Reply:
https://github.com/iterative/terraform-provider-iterative/pull/471/commits/060dc4341cc9d20a0af59a55f4f4d569bddacbbc makes status consistent with the `parallelism` value; related to #357 & #389.